src/lib/storage/value_segment.hpp

```diff
@@ -32,6 +32,7 @@ class ValueSegment : public AbstractSegment {
 
  protected:
   // Implementation goes here
+  std::vector<T> _segment_data;
```

Comment:
We believe it is better style to explicitly call the default constructor at this point.